dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -892,6 +892,34 @@ export const Explorer = ({
           [GROUPBY]: [{ bucketSize: '', bucketOffset: '' }],
           [AGGREGATIONS]: [],
         };
+      case VIS_CHART_TYPES.LogsView: {
+        const dimensions = [
+          ...statsToken.aggregations
+            .map((agg) => ({
+              label: `${agg.function.name}(${agg.function.value_expression})` ?? '',
+              name: `${agg.function.name}(${agg.function.value_expression})` ?? '',
```

Comment:
You could have saved the same in a variable and assign the value to label and name. The below assignment inside the concat is fine, as that is not an expression. Here you are evaluating an expression and could have been saved in a variable.

Reply:
Resolved.